code-challenges.test.js

```diff
@@ -56,4 +116,25 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
-// b) Create the function that makes the test pass.
+// ReferenceError: indexVowel is not defined
+// Good error
+
+// b) Create the function that makes the test pass
+// First I created a function called indexVowel
+// Then we'll create a for loop with the variable i = 0, and define the condition i must be less than the length of the string with <str.length, and finally i++ which will increase a value each time the loop has been executed.
+// then a conditional if statement that if the string[i] has the a,e,i,o, or u vowels (seperated by || OR) then it will return i.
+
+
+const indexVowel = (str) => {
+    for (let i = 0; i <str.length; i++) {
+        if (
+            str[i] === "a" || 
+            str[i] === "e" ||
+            str[i] === "i" ||
+            str[i] === "o" ||
+            str[i] === "u") {
+            return i
+          }
+        }
+      }
+
```

Comment:
You rocked these challenges, Nate!  Excellent work - your tests look just right, your code is clean, and you're doing a really good job of breaking things down with pseudocode.  Solid work!